openlibrary/coverstore/warc.py

```diff
@@ -62,7 +62,7 @@ def consume_crlf():
             creation_date, record_id, content_type, {})
 
         while True:
-            line = self._file.readline().decode('utf-8')
```

Comment:
Dumb question: Are we better off keeping `_file` as a `BytesIO`? I'd like to understand the purpose of the change more.

Reply:
This is a revert of a previous PR where I went down the wrong path so I am undoing a previous mistake.